dreamy-backend/server.js

```diff
@@ -0,0 +1,58 @@
+const path = require('path');
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+
```

Comment:
Should these extra whitespaces be deleted as well?

Reply:
Removed the extra white spaces